.github/workflows/BuildDylib.yaml

```
@@ -2,14 +2,11 @@ name: Build Dylib
 
 on:
   push:
-    branches:
-      - master
-    # tags: '*'
-  release:
-    types:
-      - created
-  pull_request:
-    types: [opened, synchronize, reopened]
+    paths:
+    - 'deps/Makefile'
+    - 'deps/libtask.c'
```

My bad, it should be
```suggestion
    - 'deps/task.c'
```